### DISCUSSION TITLE
Frame starting from 0 in task page

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#4114] [Ideas]
![Capture](https://user-images.githubusercontent.com/16021947/148177570-3e390cb1-1096-432a-86c6-37f5fc4e7e04.JPG)

If i upload 3 images while creating a task, we get start frame as 0 and stop frame as 2 .
How can we display the start frame from 1 as default and stop frame as 3  i.e  display 1-3 instead of 0-2 as shown in the image ?

**Comment by nmanovic** [comment]
@keshavadk , you need to change the source code. Why do you need it?

**Reply by keshva23** [reply]
@nmanovic Ok. It was as per our project requirement, for the end user.
